css-overflow-4/Overview.bs

```diff
@@ -1070,6 +1068,91 @@ Fragmentation of Overflow: the 'continue' property</h3>
 			(such as pagination of this box itself)
 			do not cause any content to be discarded.
 
+			Additionally,
+			if the box is a [=multi-column container=],
+			any content in or after [=overflow columns=] is also not rendered,
```

Comment:
Is saying that the content is not rendered, rather than the columns / multicol lines, significant? This wording could imply that `column-rule`s between overflow columns, or in a following multicol line, *are* rendered.